chatbot/views.py

```
@@ -0,0 +1,28 @@
+
+# API view to handle chat
+def chat_api(request):
+    if request.method == "GET":
+        user_message = request.GET.get('message', '')
+        print("User message:", user_message)  # Debugging the incoming message
+        bot_response = get_bot_response(user_message)
+        print("Bot response:", bot_response)  # Debugging the bot response
+        return JsonResponse({"user_message": user_message, "bot_response": bot_response})
+
+# Chatbot UI Page
+def chat_page(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `chat_page`

(ANN201)